### DISCUSSION TITLE
Modern SharePoint search keyword issue

### DISCUSSION THREAD

**Discussion by rohitchouhan0808** [microsoft-search/pnp-modern-search#1789] [Q&A]
I am using PnP modern search in SharePoint. there are few keywords which is showing error which searching for them.
I am getting the result if adding additional character or trimming the  keyword or using ' * '.
I am getting the below error code 
[HTTP]:500 - Internal Server Error [CorrelationId]:9bbd1ca0-00b2-c000-b168-21b724942f83 [Version]:16.0.0.22112

Kindly help with this issue, attached the screenshot
<img width="750" alt="Screenshot 2022-02-01 155516" src="https://user-images.githubusercontent.com/98877439/152125523-c81dbe16-f8b6-4abc-905d-f72a867da9fb.png">
.

**Comment by wobba** [comment]
Move to discussions are there is no details around your setup and repro. But seems like an API issue, similar to what others have reported. You should try to do a repro using classic SharePoint web parts for your setup, and if you get the same error, then log a support ticket with Microsoft.

**Reply by rohitchouhan0808** [reply]
We has similar setup in classic SharePoint and its working fine there, I created a ticket with Microsoft and their team suggested to raise issue in GitHub community

**Reply by wobba** [reply]
Then you need to have better repro data. API you use, query rules/result sources involved, multi-geo, etc.

If the JSON request looks valid then it's an API issue why it fails.
